src/scenes/GameScene.ts

```diff
@@ -8,13 +8,27 @@ const officeTilesetKey: string = 'office_tiles';
 const roomTilesetName: string = 'Room';
 const officeTilesetName: string = 'Office';
 
+const MAP: Record<string, string> = {
+  'office-map': 'maps/office.json',
+};
+
+const MUSIC: Record<string, string[]> = {
+  'office-map': ['audio/route1.mp3'],
+};
+
+/**
+ * A game / playable scene, this scene would have a player that moves around a map.
+ */
 export default class GameScene extends Phaser.Scene {
```

Comment:
❓ Do we need to make this abstract if this is going to be the "base class"?

Reply:
We dont need to, but would help us from mistakingly creating an instance.